docker/local-universe/Dockerfile.static

```diff
@@ -0,0 +1,30 @@
+## The versions in this file are hardcoded for audit purposes, and should not be changed.
```

Comment:
If we do need to change these (say, due to a security vulnerability), what are the consequences? Do we need to notify customers?